Date: Sat, 1 May 2004 02:28:28 +1000
From: Anton Blanchard <anton@samba.org>
To: olh@suse.de
Subject: [anton@samba.org: [PATCH] [ppc64] ARCH_HAS_SCHED_WAKE_BALANCE doesnt exist]


Hi,

A bug found in # 7981

Anton

----- Forwarded message from Anton Blanchard <anton@samba.org> -----

From: Anton Blanchard <anton@samba.org>
To: akpm@osdl.org
Subject: [PATCH] [ppc64] ARCH_HAS_SCHED_WAKE_BALANCE doesnt exist


Hi,

It seems someone has been making trivial changes without using grep.

Anton

--- linux-2.6.5/include/asm-ppc64/processor.h~	2004-04-30 11:10:36.056563140 -0500
+++ linux-2.6.5/include/asm-ppc64/processor.h	2004-04-30 11:10:59.565459656 -0500
@@ -630,7 +630,7 @@
 
 #ifdef CONFIG_SCHED_SMT
 #define ARCH_HAS_SCHED_DOMAIN
-#define ARCH_HAS_SCHED_WAKE_BALANCE
+#define ARCH_HAS_SCHED_WAKE_IDLE
 #endif
 
 #endif /* ASSEMBLY */

----- End forwarded message -----

